### PR TITLE
Plans for Metazoa e114

### PR DESCRIPTION
**Pull request by JAlvarezJarreta**
## Description

Updates for all 3 gene trees planned for e114, as discussed during the past few months, due to the update of _Drosophila melanogaster_. We have also taken the opportunity to reduce the overlap between metazoa/protostomes and insects collection, leaving only _D. melanogaster_ as the single species overlap.

**Related JIRA tickets:**
- ENSINT-1910
- ENSINT-1911

### CODE REVIEW

File: conf/metazoa/mlss_conf.xml [hunk 1]
```diff
@@ -181,127 +187,129 @@
...
-      <genome name="stomoxys_calcitrans"/>
-      <genome name="teleopsis_dalmanni"/>
-      <genome name="thrips_palmi_gca012932325v1rs"/>
-      <genome name="trialeurodes_vaporariorum_gca011764245"/>
-      <genome name="tribolium_castaneum"/>
-      <genome name="trichogramma_pretiosum_gca000599845v3"/>
-      <genome name="venturia_canescens_gca019457755v1rs"/>
-      <genome name="zerene_cesonia_gca012273895v2rs"/>
-      <genome name="zootermopsis_nevadensis"/>
+      <genome name="acromyrmex_echinatior_gca000204515v1rs">
```
Comment (vsitnik) [inline conf/metazoa/mlss_conf.xml:190]: no closing `/` . is it as expected?

**Review by vsitnik — CHANGES_REQUESTED** [review]
I think, we miss a closing `/`  in the xml part.

File: conf/metazoa/mlss_conf.xml [hunk 2]
```diff
@@ -181,127 +187,129 @@
...
+      <genome name="solenopsis_invicta">
+      <genome name="stomoxys_calcitrans">
+      <genome name="teleopsis_dalmanni">
+      <genome name="thrips_palmi_gca012932325v1rs">
+      <genome name="trialeurodes_vaporariorum_gca011764245">
+      <genome name="tribolium_castaneum">
+      <genome name="trichogramma_pretiosum_gca000599845v3">
+      <genome name="venturia_canescens_gca019457755v1rs">
+      <genome name="zerene_cesonia_gca012273895v2rs">
+      <genome name="zootermopsis_nevadensis">
```
[the remark this answers is not in the document]
Comment (JAlvarezJarreta) [reply conf/metazoa/mlss_conf.xml:312]: 🤦‍♂️ thank you!

**Review by vsitnik — APPROVED** [review]
Thank you!

File: conf/metazoa/mlss_conf.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (vsitnik) [reply conf/metazoa/mlss_conf.xml]: Thank you, @twalsh-ebi!

**Review by Simarpreet-Kaur-Bhurji — APPROVED** [review]

**Pull request merged**